docs/components/tabs/index.html

```diff
@@ -11,99 +11,104 @@
 {% extends 'component.njk' %}
 {% block content %}
   <section>
-    <h2 id="basic-tabs">Basic Tabs</h2>
-    <div class="demo tabs-demo" data-visreg="tabs">
-      <hx-tabset id="android-desserts">
-        <hx-tablist>
-          <hx-tab>Cupcake Ipsum</hx-tab>
-          <hx-tab id="marshmallow">Biscuit Marshmallow</hx-tab>
-          <hx-tab>Caramels Marzipan</hx-tab>
-        </hx-tablist>
-        <hx-tabcontent>
-          <hx-tabpanel>
-            <h3>First Panel</h3>
-            <div class="hxRow">
-              <div class="hxCol hxSpan-6">
-                <p>
-                  Cupcake ipsum dolor sit amet bonbon topping caramels. Sesame snaps
-                  gummi bears liquorice cookie chupa chups fruitcake croissant
-                  chocolate topping. Brownie biscuit wafer marshmallow liquorice
-                  soufflé powder jelly.
-                </p>
+    <h2 id="basic-tabset">Basic Tabset</h2>
+    <div class="example">
+      <div class="tabs-demo" data-visreg="tabs">
```

Comment:
Do we still need the `data-visreg` attribute here?

Reply:
yes, because we haven't removed visreg tests yet